src/CQRS/LeanCode.CQRS.OutputCaching/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,49 @@
+using System.Reflection;
+using LeanCode.Components;
+using LeanCode.CQRS.Execution;
+using LeanCode.CQRS.OutputCaching.Registration;
+using Microsoft.AspNetCore.OutputCaching;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace LeanCode.CQRS.OutputCaching;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddCQRSOutputCache(
+        this IServiceCollection services,
+        TypesCatalog catalog,
+        bool registerOutputCache = true
+    )
+    {
+        return services.AddCQRSOutputCache(catalog.Assemblies, registerOutputCache);
+    }
+
+    public static IServiceCollection AddCQRSOutputCache(
+        this IServiceCollection services,
+        IEnumerable<Assembly> assemblies,
```

Comment:
The `TypesCatalog` one should be the only overload. We introduced that type to _not_ have to deal with raw assemblies.